Framework/PythonInterface/plugins/algorithms/SaveYDA.py

```diff
@@ -0,0 +1,314 @@
+from mantid.api import PythonAlgorithm, AlgorithmFactory, MatrixWorkspaceProperty, WorkspaceUnitValidator, \
```

Comment:
Can we include `from __future__ import absolute_import, division, print_function` as the first line in the file?

We are trying to make all of our Python code 2/3 compatible and having this line helps ensure that for the future.